transports/msmq/index.md

```diff
@@ -71,12 +71,14 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
 ## Permissions
 
-| Group          | Permissions          |
-|----------------|----------------------|
-| Owning account | Write, Receive, Peek |
-| Administrators | Full                 |
-| Anonymous      | Write                |
-| Everyone       | Write                |
+| Group          | Permissions         | Description                       |
+|----------------|---------------------|-----------------------------------|
+| Owning account | Send, Receive, Peek | Set by NServiceBus                |
+| Administrators | Full                | Set by NServiceBus                |
+| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
```

Comment:
@indualagarsamy Its part of https://github.com/Particular/NServiceBus/releases/tag/6.1.0

> #4304 Anonymous permissions added to existing MSMQ queues

Those hotfixes we that we would set permissions always, not only at creation, overriding permissions set by the user. Maybe we should add that as a note too?